sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -70,9 +76,11 @@ class EventHubConsumerClient(ClientBase):
      keys: `'proxy_hostname'` (str value) and `'proxy_port'` (int value).
      Additionally the following keys may also be present: `'username', 'password'`.
     :keyword checkpoint_store: Manager for storing the partition load-balancing and checkpoint data when receiving
-     events. If None, this `EventHubConsumerClient` instance will receive events without load-balancing and
-     checkpointing. The checkpoint store will be used in both cases of receiving from all partitions or a single
+     events. If not provided, `update_checkpoint` method `PartitionContext` object could still be called for in-memory
```

Comment:
Not quite sure what this is saying.... perhaps something like this?
"For in-memory checkpointing operation, updating the checkpoint could (will? should?) happen automatically regardless of whether explicitly called in the supplied handler.".

Also - replace comma with period at end of line 80.

This comment also applies to the same doc change on line 188.

Reply:
In current implementation, if the user doesn't provide a checkpoint_store, we'd internally use a InMemoryCheckpointStore which is implicit to user.

So previously when user call update_checkpoint without giving checkpoint_store, we would log an warning.
Now the update_checkpoint can work without a user provided checkpoint_store.

Maybe something like:

When the checkpoint_store is not provided, the SDK will use a InMemoryCheckpointStore internally which can do in-memory checkpoint .

Reply:
In current implementation, if the user doesn't provide a checkpoint_store, we'd internally use a InMemoryCheckpointStore which is implicit to user.

So previously when user call update_checkpoint without giving checkpoint_store, we would log an warning.
Now the update_checkpoint can work without a user provided checkpoint_store.

Maybe something like this?
```
When the checkpoint_store is not provided, the SDK will use a InMemoryCheckpointStore internally which can do in-memory checkpoint .
```

Or do you have any better ideas..

Reply:
Ah - in that case I would go with something like:
```
If a checkpoint store is not provided, the checkpoint will be maintained internally in memory.
```